packages/syft/src/syft/service/code/user_code.py

```diff
@@ -847,24 +869,177 @@ class UserCodeExecutionResult(SyftObject):
     result: Any
 
 
-def execute_byte_code(code_item: UserCode, kwargs: Dict[str, Any]) -> Any:
+def execute_byte_code(
+    code_item: UserCode, kwargs: Dict[str, Any], context: AuthedServiceContext
+) -> Any:
     stdout_ = sys.stdout
     stderr_ = sys.stderr
 
     try:
+        # stdlib
+        import builtins as __builtin__
+
+        original_print = __builtin__.print
+
+        class LocalDomainClient:
+            def __init__(self):
+                pass
+
+            def launch_job(self, func: UserCode, **kwargs):
+                # original_print("launching job for ", func.service_func_name)
+                # relative
+                from ... import UID
+
+                # get reference to node (TODO)
+                node = context.node
```

Comment:
this is obviously the dangerous thing here. I am not sure how easy it would be to get a reference to this node object from within the syft function, but once you do you can basically do anything server side